### DISCUSSION TITLE
post/running-microservices-aca-dapr

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#54] [Announcements]
# post/running-microservices-aca-dapr

Mark Heath's Development Blog

https://markheath.net/post/running-microservices-aca-dapr

**Comment by twiga2013** [comment]
Hi Mark, can Dapr be hosted in Azure App service and be used from Azure Functions?

**Reply by markheath** [reply]
hosting Dapr on regular Windows Azure App Service is not possible as it doesn't offer a way to run a "sidecar" alongside your app which is what Dapr needs. I think it might be possible with the Linux App Service which is container based, but really I'd look into Azure Container Apps as a better hosting platform for Dapr. ACA also supports KEDA, so you should be able to combine the Azure Functions programming model with Dapr (although its not something I've tried myself yet).
